### DISCUSSION TITLE
Golang优雅之道 - Qinng

### DISCUSSION THREAD

**Discussion by giscus** [qingwave/qingwave.github.io#78] [Announcements]
# Golang优雅之道 - Qinng

借助一些设计模式、流式编程、函数编程的方法可以让我们的Golang代码更清晰优雅，本文中描述了在错误处理、可选配置、并发控制等方面的优化手段。 链式错误处理很多人不喜欢Go的错误处理，需要写大量if err !=

https://qingwave.github.io/golang-programming-pattern/

**Comment by acechef** [comment]
强强强
